### PR TITLE
chore: add start and watch run-scripts

### PR DESCRIPTION
**Pull request by marionebl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,20 +9,22 @@
...
-    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -w",
     "preversion": "npm test",
+    "start": "npm run watch",
+    "test": "ava",
     "version": "npm run build && conventional-changelog -p angular -v -r 0 > CHANGELOG.md && git add -A .",
-    "postversion": "git push && git push --tags",
-    "prepublish": "not-in-install && npm run build || echo 'In install cycle, skipping prepublish'",
-    "precoverage": "npm run clean && tsc --inlineSourceMap",
-    "coverage": "nyc --reporter lcov ava"
+    "watch": "npm run test -- --watch"
```
Comment (KnisterPeter) [inline package.json:27]: This is not sufficent. Devs would expect to start a full watch process. This only watches the tests.
You still need to build the sources (and probably what the build process).

**Comment by KnisterPeter** [comment]
Why did you reorder all the scripts? They was ordered by case (e.g. 'prestep', 'step', 'poststep')

**Comment by marionebl** [comment]
I figured alphabetical order makes most sense. Can change this if you have a strong opinion on this.

**Comment by marionebl** [comment]
Superseeded by #13

**Pull request closed without merging**